### PR TITLE
Make README.md more clean

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ I have an open instance hosted at https://aupro.xyz.
 
 This software should be used for personal use on public lobbies. Anyone that wants to
 have a custom solution, private server, or other commercial use for this software should
-send a direct message to `edward#2222` on Discord.
+send a direct message to <a href="https://discordapp.com/users/508426414387757057">`edward#2222`</a> on Discord.
```
Comment (edqx) [inline README.md]: What does this do? Also, `discord.com` is the new domain.

**Review by edqx — COMMENTED** [review]
Thanks, I hadn't realised that this could be improved.

File: README.md (hunk 1 above)
Comment (MatadorProBr) [reply README.md]: Fixed

Comment (ghost) [reply README.md]: Fixed

**Pull request merged**